src/main/java/com/book/booksstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.book.booksstore.service;
+
+import com.book.booksstore.model.Book;
+import com.book.booksstore.repository.BookRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
if you have some issues with RequiredArgsConstructor - please text in chat
```suggestion
@Service
@RequiredArgsConstructor
```

Reply:
added